### PR TITLE
Add fragment block

### PR DESCRIPTION
**Pull request by Matei-Iordache**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--helix-poc--enake.hlx.page/consumer/en/test
- After: https://fragment-block--helix-poc--enake.hlx.page/consumer/en/test

### CODE REVIEW

**Review by Tiseanu — APPROVED** [review]

File: blocks/fragment/fragment.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import {
+  loadBlocks,
+} from '../../scripts/lib-franklin.js';
+
+/**
+   * Loads a fragment.
+   * @param {string} path The path to the fragment
+   * @returns {HTMLElement} The root element of the fragment
+   */
+async function loadFragment(path) {
```
Comment (vladalexandruradulescu) [inline blocks/fragment/fragment.js:20]: let's add a e2e test for this one

**Review by vladalexandruradulescu — CHANGES_REQUESTED** [review]

**Review by Matei-Iordache — COMMENTED** [review]
This has been discussed and is no longer needed

**Review by Matei-Iordache — COMMENTED** [review]
sas

**Review by Matei-Iordache — COMMENTED** [review]
lgtm

File: blocks/fragment/fragment.js (hunk 1 above)
Comment (Matei-Iordache) [reply blocks/fragment/fragment.js:21]: solved

**Pull request merged**